middleware/ratelimiter/ratelimiter_test.go

```diff
@@ -0,0 +1,111 @@
+package ratelimiter
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/System-Glitch/goyave/v3"
+)
+
+type RateLimiterMiddlewareTestSuite struct {
+	goyave.TestSuite
+}
+
+func TestRateLimiterMiddlewareTestSuite(t *testing.T) {
+	goyave.RunTest(t, new(RateLimiterMiddlewareTestSuite))
+}
+
+func (suite *RateLimiterMiddlewareTestSuite) TestLimiterResponseHeaders() {
+
+	requestQuota := 10
+	numberOfRequests := 7
+	quotaDuration := 5 * time.Second
+
+	ratelimiterMiddleware := New(func(request *goyave.Request) LimiterConfig {
+		return LimiterConfig{
+			RequestQuota:  requestQuota,
+			QuotaDuration: quotaDuration,
+		}
+	})
+
+	request := suite.CreateTestRequest(nil)
+
+	var result *http.Response
+
+	for i := 0; i < numberOfRequests; i++ {
+		result = suite.Middleware(
+			ratelimiterMiddleware,
+			request,
+			func(response *goyave.Response, request *goyave.Request) {},
+		)
+	}
+
+	suite.Equal(
+		fmt.Sprintf("%v, %v;w=%v", requestQuota, requestQuota, quotaDuration.Seconds()),
+		result.Header.Get("RateLimit-Limit"),
+	)
+
+	suite.Equal(
+		fmt.Sprintf("%v", requestQuota-numberOfRequests),
+		result.Header.Get("RateLimit-Remaining"),
+	)
```

Comment:
`RateLimite-Reset` header's value is not tested.

Reply:
Oh yes! I have been trying to figure out how to test it.

I would like to get your input on this.

Reply:
- Get the remote address from the request created with `suite.CreateTestRequest`.
- Use it to get the instance of `limiter` in your store.
- Inject your own newly generated time so you can compare easily without the risk of time difference due to execution time.